src/tests/e2etest/CommandRunnerTests.cs

```diff
@@ -39,54 +43,219 @@ public partial class CommandArguments
             public bool SingleLineTextResult { get; set; }
         }
 
+        public partial class CommandArgumentsResponse
+        {
+            public CommandArguments value { get; set; }
+            public int ac { get; set; }
+            public string ad { get; set; }
+            public int av { get; set; }
```

Comment:
What is the test doing with these?

Reply:
Test is checking `ac` (acknowledge code) and the others are there to "fill out" the rest of the JSON object in the twin

Reply:
Cool. However, how are they filled?
av for example is version.
OSConfig sets that to same version that comes in the request. Same version goes in the acknowledge.
These should not be set blindly. May produce unexpected behavior with the Reported Twin.

Reply:
After looking at this, we might want to bring this into the base `E2eTest` base class. We use this for brokering the twin updates, so it makes sense to bring this into all tests.
I would suggest adding this to `UpdateTwinBlockUntilUpdate`?

Reply:
Removed `ad` and `av`. Will move this to E2eTest base class when refactoring/adding E2E test tooling